_includes/feature-phases/preview-opt-in.md

```diff
@@ -0,0 +1,3 @@
+{{site.data.alerts.callout_info}}
+**This feature is in preview**, and only available to enrolled organizations. To enroll your organization in the preview, contact your Cockroach Labs account team. To share feedback and/or issues, contact [Support](https://support.cockroachlabs.com/hc/en-us).
```

Comment:
Nit: I don't think you need the comma before the first "and".

"The interface and output are subject to change" might not apply to every feature. I wonder if this would be better as "Where applicable, API interfaces and output related to this feature are subject to change." (if that's what you mean).

Related to a specific private preview feature that is already documented, we were asked not to invoke Support as the first port of call for feedback. I wonder if there is broad agreement on this yet.